lib/runtime/procs/pace.js

```diff
@@ -116,15 +116,17 @@ class pace extends fanin {
         }
     }
     mark(mark) {
+        mark.time = this.offset_time(mark.time);
```

Comment:
You should make a copy before modifying in order to avoid influencing other procs working with the mark (e.g. in a parallel stream).